migrations/versions/e46b30dd3ff5_automatic_collections.py

```diff
@@ -0,0 +1,65 @@
+"""automatic_collections
+
+Revision ID: e46b30dd3ff5
+Revises: 9e33fa16ba82
+Create Date: 2023-02-06 11:44:29.000880
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+
+# revision identifiers, used by Alembic.
+revision = 'e46b30dd3ff5'
+down_revision = '9e33fa16ba82'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # TODO: Maybe add some CHECK constraints around some of these fields? Or just
+    # enforce in code upon creation.
+    op.create_table(
+        "automatic_collection_definition",
```

Comment:
I suggest changing the name of this table to "automatated_collections" just to shorten it and make it clearer.